From ecb616f1628ac740c052ea29ee730382f1441dda Mon Sep 17 00:00:00 2001
From: Scott K Logan <logans@cottsay.net>
Date: Thu, 28 Jan 2021 14:33:37 -0800
Subject: [PATCH] Disable sanitizers in RPM build

Signed-off-by: Scott K Logan <logans@cottsay.net>
---
 rpm/template.spec.em | 1 +
 1 file changed, 1 insertion(+)

diff --git a/rpm/template.spec.em b/rpm/template.spec.em
index d2872c68..8cdee61b 100644
--- a/rpm/template.spec.em
+++ b/rpm/template.spec.em
@@ -46,6 +46,7 @@ mkdir -p obj-%{_target_platform} && cd obj-%{_target_platform}
     -DAMENT_PREFIX_PATH="@(InstallationPrefix)" \
     -DCMAKE_PREFIX_PATH="@(InstallationPrefix)" \
     -DSETUPTOOLS_DEB_LAYOUT=OFF \
+    -DDISABLE_SANITIZERS=ON \
     ..
 
 %make_build
-- 
2.17.1

