From 3b4249f93cc5740b0b77d46e54b773006975826b Mon Sep 17 00:00:00 2001
From: Scott K Logan <logans@cottsay.net>
Date: Fri, 18 Feb 2022 10:47:45 -0800
Subject: [PATCH] Disable sanitizers in RPM build

---
 rpm/template.spec.em | 1 +
 1 file changed, 1 insertion(+)

diff --git a/rpm/template.spec.em b/rpm/template.spec.em
index 5d2afb373..4256d15f5 100644
--- a/rpm/template.spec.em
+++ b/rpm/template.spec.em
@@ -54,6 +54,7 @@ mkdir -p .obj-%{_target_platform} && cd .obj-%{_target_platform}
 %if !0%{?with_tests}
     -DBUILD_TESTING=OFF \
 %endif
+    -DDISABLE_SANITIZERS=ON \
     ..
 
 %make_build
-- 
2.43.0

